index2.js

```diff
@@ -0,0 +1,2 @@
+111111111;
+sdasdadads;
```

Comment:
너무 길어요. 이렇게 수정하는건 어떨까요?
```suggestion
111;
ads;
```

Reply:
@KumJungMin **오**

Reply:
@KumJungMin 오

Reply:
a98f701 수정했습니다~